File: atv2/exerc2.py
class Carro:
    def __init__(self, marca, modelo, ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano
        self.velocidade = 0

    def acelerar(self):
        self.velocidade += 10

    def frear(self):
        self.velocidade -= 10

    def exibir_velocidade(self):
        print(f"Velocidade atual: {self.velocidade}")

c = Carro("Ford", "Fusion", 2020)
c.acelerar()
c.exibir_velocidade()
c.frear()
c.exibir_velocidade()

File: atv2/exerc2.go
package main
import "fmt"

type Carro struct {
    Marca      string
    Modelo     string
    Ano        int
    Velocidade int
}

func (c *Carro) acelerar() {
    c.Velocidade += 10
}

func (c *Carro) frear() {
    c.Velocidade -= 10
}

func (c Carro) exibirVelocidade() {
    fmt.Println("Velocidade atual:", c.Velocidade)
}

func main() {
    c := Carro{"Ford", "Fusion", 2020, 0}
    c.acelerar()
    c.exibirVelocidade()
    c.frear()
    c.exibirVelocidade()
}
